src/framework/ui/toggle/toggle.component.tsx

```diff
@@ -133,44 +133,46 @@ export class Toggle extends React.Component<Props> {
   };
 
   private getComponentStyle = (style: StyleType): StyleType => {
-    const { thumb, highlight, ...container } = style;
-    const thumbComponentSize: number = container.height - (container.borderWidth * 2);
+    const thumbComponentSize: number = style.height - (style.borderWidth * 2);
 
     return {
       wrapper: {
-        width: container.width,
-        height: container.height,
+        width: style.width,
+        height: style.height,
       },
       componentContainer: {
-        ...container,
+        ...style,
```

Comment:
really? all styles to one container, but some of them to `wrapper`?

Reply:
Refactored, it was just an unused code.